src/commands/purge.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Purge a QuantCDN url.
```

Comment:
```suggestion
 * Purge the cache for a given url.
```

Purging a URL sounds like it's destructive (rather than just a cache flush)